tests/advanced_audit/run_complete_audit.py

```
@@ -47,11 +47,12 @@ def __init__(self, config_path: str = "config.yaml"):
     
     def _load_config(self, config_path: str) -> Dict[str, Any]:
         """Load YAML configuration"""
+        # noqa: PT - config_path is hardcoded at call site, not user input
         config_file = Path(__file__).parent / config_path
         if not config_file.exists():
             raise FileNotFoundError(f"Config file not found: {config_file}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)